lld/COFF/DriverUtils.cpp

```diff
@@ -862,6 +862,9 @@ opt::InputArgList ArgParser::parse(ArrayRef<const char *> argv) {
                 << "', did you mean '" << nearest << "'";
   }
 
+  if (args.hasArg(OPT_link))
+    Warn(ctx) << "ignoring /link, did you pass it multiple times?";
```

Comment:
Is there any value for this message? We can silently ignore the flng and nothing bad will happen.

Reply:
I did this to match link.exe which gives a warning.